content/en/blog/_posts/2023-01-15-Security-Bahavior-Analysis/index.md

```diff
@@ -0,0 +1,86 @@
+
+The diagram above clarifies how dividing a monolithic service to a set of microservices improves our ability to perform security-behavior monitoring and control. In a monolithic service approach, different client requests are intertwined, resulting in diminished ability to identify irregular client behaviors. Without prior knowledge, an observer of the intertwined client requests will find it hard to distinguish between types of requests and their related characteristics. Further, internal client requests are not exposed to the observer. Lastly, the aggregated behavior of the monolithic service is a compound of the many different internal behaviors of its components, making it hard to identify irregular service behavior.
+
+In a microservice environment, each microservice is expected by design to offer a more well-defined service and serve better defined type of requests. This makes it easier for an observer to identify irregular client behavior and irregular service behavior. Further, a microservice design exposes the internal requests and internal services which offer more security-behavior data to identify irregularities by an observer. Overall, this makes the microservice design pattern better suited for security-behavior monitoring and control.
+
+## Security-Behavior Under Kubernetes
+
+Kubernetes deployments seeking to add Security-Behavior may use [Guard](http://knative.dev/security-guard), developed under the CNCF project Knative. **Guard can be installed as a standalone tool to protect any HTTP-based Kubernetes service**. Alternatively, Guard is integrated into the full Knative automation suite that runs on top of Kubernetes.
+
+See:
+
+- [Guard on Github](https://github.com/knative-sandbox/security-guard) for using Guard as a standalone.
+- The Knative automation suite - Read about Knative, as an [Opinionated Kubernetes](https://davidhadas.wordpress.com/2022/08/29/knative-an-opinionated-kubernetes) implementation that simplifies and unifies the way web services are deployed on Kubernetes.
+- You can also find more information on how to [secure microservices by monitoring behavior](https://developer.ibm.com/articles/secure-microservices-by-monitoring-behavior/)
+
+The goal of this post is to invite the Kubernetes community to action and introduce Security-Behavior monitoring and control to help secure Kubernetes based deployments. Hopefully, the community as a follow up will:
+
+1. Analyze the cyber challenges presented for different Kubernetes use cases
+1. Add appropriate security documentation for users on how to introduce Security-Behavior monitoring and control.
+1. Consider how to integrate with tools that can help users monitor and control their vulnerable services.
+
+## Getting Involved
+
+You are welcome to get involved and join the effort to develop security behavior monitoring
```

Comment:
Not sure if I asked this earlier, but is there a link to a `Getting Started Guide` or a slack channel / mailing list or other place where people can actually go to _get involved_ ? Right now this section encourages involvement without clear guidance on `how`.

Reply:
ok, I will see what I can add

Reply:
I added a link to the slack channel of the maintainers (it is not being moved to cncf)

Reply:
> Not sure if I asked this earlier, but is there a link to a Getting Started Guide or a slack channel / mailing list or other place where people can actually go to get involved ? Right now this section encourages involvement without clear guidance on how.

There is a Getting Started page on kubernetes.dev
https://www.kubernetes.dev/docs/guide/#getting-started